### ISSUE TITLE
Internal server error 500 when opening public links

### ISSUE THREAD

**Issue by MSBack** [Sekund/sekund-plugin-react#29]
I cannot get the public links working, they only return an internal server error 500.

**Comment by ckemmler** [comment]
Yes there is mainly the problem that the publishing engine expects jsx so if you have custom html or styles in your markdown, it could constitute a choke point. Will be working on a fix as soon as back from holidays. Le vendredi 19 août 2022 à 13:36, MSBack ***@***.***> a écrit :  
I cannot get the public links working, they only return an internal server error 500.
